### PR TITLE
Remove POT-Creation-Date as part of make-translations.sh

### PR DESCRIPTION
**Pull request by dannyroberts**
## Technical Summary
Remove POT-Creation-Date as part of make-translations.sh

It changes every time there are any translations and so is conflict prone.
I don't think it provides any value.

### Safety story
I will add just a little more research about what POT-Creation-Date is for and run update-translations.sh to see if not having it interferes.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by joxl** [comment]
Correct me if I'm wrong, but it appears that _all_ `POT-Creation-Date` values get re-generated _every time_. If this is true then I think it's fair to say they are effectively useless.  I had a look at the [transfex gettext docs](https://docs.transifex.com/formats/gettext) which suggest that these values are all just "developer comments" anyway, further suggesting that this value is safe to omit.

**Review by esoergel — APPROVED** [review]
Thanks!  Yeah, since we store this in a git repo, that timestamp really doesn't provide us with any useful info.  No idea whether transifex uses it for something, though.

**Comment by dannyroberts** [comment]
Seems to apply fine, no issues. I'll fix the merge conflict (last one of these ever!) and merge

File: scripts/make-translations.sh [hunk 1]
```diff
@@ -24,14 +24,9 @@ if ! ./manage.py compilemessages; then
...
+# Remove POT-Creation-Date which changes every time and is conflict-prone
 git diff --name-only | grep '^locale/' | while read -r filename
 do
-    if ! git diff -U0 -- "$filename" | tail -n +6 | grep -v '^[+-]"POT-Creation-Date:\s' > /dev/null
-    then
-        # disable hooks for this command
-        git -c core.hooksPath=/dev/null checkout -- "$filename"
-    fi
+    sed -i.bak '/^"POT-Creation-Date: .*"$/d' $filename
+    rm ${filename}.bak
```
Comment (joxl) [inline scripts/make-translations.sh:31]: nit (sed's `-i` is a pain)
```suggestion
    sed -i '' '/^"POT-Creation-Date: .*"$/d' $filename
```

Comment (dannyroberts) [reply scripts/make-translations.sh:31]: Ah! Thanks, I forgot you could use `-i` without backup. Thanks, this is much better.

**Review by joxl — APPROVED** [review]

**Comment by dannyroberts** [comment]
Error is `sed: can't read /^"POT-Creation-Date: .*"$/d: No such file or directory` which makes me think the version of `sed` installed in the test VM is somehow different from what I have locally, and is interpreting the expression arg as a filename arg.

**Comment by dannyroberts** [comment]
I think maybe the problem is how the different versions of `sed` treat the zero-length extension (expecting `-i` vs `-i ''`). I'm going to try going back to what I had before and seeing if that fixes it.

**Comment by joxl** [comment]
> different from what I have locally

Ah, yes. Use `-e` to explicitly tell it "this is my pattern", I think that will work:

```shell
sed -i '' -e '/^"POT-Creation-Date: .*"$/d' "$filename"
```

**Comment by dannyroberts** [comment]
Okay my original clunky version passed the tests, let's try your -e suggestion @joxl!

**Comment by joxl** [comment]
Ugh, I have no idea @dannyroberts. Go with the one that writes and `rm`s an extra file. Sorry for the hassle.

:subdued:

**Pull request merged**